projects/mirza-test-utils-haskell/mirza-test-utils-haskell.cabal

```
@@ -0,0 +1,74 @@
+cabal-version: 1.12
+
+-- This file has been generated from package.yaml by hpack version 0.31.2.
+--
+-- see: https://github.com/sol/hpack
+--
+-- hash: 6aa3d2cfc3680fe1939163460b1a61264abe30d35c0b80b49229e68ab7c1f8c9
+
+name:           mirza-test-utils-haskell
+version:        0.1.0.0
+description:    Please see the README on GitHub at <https://github.com/data61/Mirza/projects/mirza-test-utils-haskell#readme>
+homepage:       https://github.com/data61/Mirza
+bug-reports:    https://github.com/data61/Mirza/issues
+author:         Author name here
+maintainer:     example@example.com
+copyright:      2017, 2018, 2019 CSIRO
+license:        BSD3
+license-file:   LICENSE
+build-type:     Simple
+extra-source-files:
+    README.md
+
+source-repository head
+  type: git
+  location: https://github.com/data61/Mirza
+
+library
```

```suggestion
library
  default-extensions:
      OverloadedStrings
```
It's a fairly safe and almost always necessary extension, so it's worth turning on by default.